application/forms/Security/RoleForm.php

```diff
@@ -262,7 +262,7 @@ public function createInsertElements(array $formData = array())
                                     PREG_SPLIT_DELIM_CAPTURE|PREG_SPLIT_NO_EMPTY
                                 )))),
                             'description'   => $spec['description'],
-                            'style'         => $isUnrestricted ? 'text-decoration:line-through;' : '',
+                            'class'         => $isUnrestricted ? 'role-restriction-text' : '',
```

Comment:
I don't think `role-restriction-text` alone is quite appropriate for its intended use. I suggest changing this to something like `role-restriction unrestricted`.

Reply:
Replaced with the name `unrestricted-role`